packages/mcp-server/src/mcp/utils.ts

```diff
@@ -16,16 +17,34 @@ export async function fetch_with_timeout(
 	}
 }
 
+const summaries = (summaryData.summaries || {}) as Record<string, string>;
+
 export async function get_sections() {
 	const sections = await fetch_with_timeout(
 		'https://svelte.dev/docs/experimental/sections.json',
 	).then((res) => res.json());
 	const validated_sections = v.safeParse(documentation_sections_schema, sections);
 	if (!validated_sections.success) return [];
-	return Object.entries(validated_sections.output).map(([, section]) => ({
-		title: section.metadata.title,
-		use_cases: section.metadata.use_cases ?? 'read document for use cases',
-		slug: section.slug,
-		url: `https://svelte.dev/${section.slug}/llms.txt`,
-	}));
+	return Object.entries(validated_sections.output).map(([, section]) => {
+		const original_slug = section.slug;
+		const cleaned_slug = original_slug.startsWith('docs/')
+			? original_slug.slice('docs/'.length)
+			: original_slug;
+
+		return {
+			title: section.metadata.title,
+			use_cases:
+				section.metadata.use_cases ?? summaries[original_slug] ?? summaries[cleaned_slug] ?? '',
```

Comment:
```suggestion
				section.metadata.use_cases ?? summaries[original_slug] ?? summaries[cleaned_slug] ?? 'read document for use cases',
```

Reply:
I don't like "read document for use cases" because it instructs the model to **read** the document, and we don't actually want that, we want the model to guess if it needs the document based on the info it has (title & path)

Reply:
Yeah maybe it's better
```suggestion
				section.metadata.use_cases ?? summaries[original_slug] ?? summaries[cleaned_slug] ?? 'use title and path to estimate use case',
```
?